source/index.txt

```diff
@@ -34,20 +32,81 @@ a Confluent-verified connector that persists data from Kafka topics as a
 data sink into MongoDB as well as publishes changes from MongoDB into Kafka
 topics as a data source.
 
-This guide provides information on available configuration options and
-examples to help you complete your implementation in the following
-sections:
-
-- :doc:`What's New </whats-new>`
-- :doc:`Quick Start </quick-start>`
-- :doc:`Introduction </introduction>`
-- :doc:`Tutorials </tutorials>`
-- :doc:`Sink Connector </sink-connector>`
-- :doc:`Source Connector </source-connector>`
-- :doc:`Security and Authentication </security-and-authentication>`
-- :doc:`Monitoring </monitoring>`
-- :doc:`Migrate from the Community Connector </migrate-from-kafka-connect-mongodb>`
-- :doc:`Troubleshooting </troubleshooting>`
-- :doc:`How to Contribute </contribute>`
-- :doc:`Issues & Help </issues-and-help>`
-- :doc:`Compatibility </compatibility>`
+What's New
+----------
+
+For a list of new features and changes in each version, see the
+:ref:`kafka-connector-whats-new` section.
+
+Quick Start
+-----------
+
+Learn how to get started with the connector and begin working with data
+in the :ref:`kafka-quick-start` section.
+
+Introduction
+------------
+
+Learn about key concepts and processes related to the connector in the
+:ref:`kafka-connector-introduction` section.
+
+Tutorials
+---------
+
+Learn how to set up and perform specific tasks with the connector
+in the :ref:`kafka-connector-tutorials` section.
+
+Sink Connector
+--------------
+
+Learn how to read data from {+kafka+} and write data to MongoDB
+with the sink connector in the :ref:`kafka-sink-overview` section.
+
+Source Connector
+----------------
+
+Learn how to read data from MongoDB and write data to {+kafka+}
+with the source connector in the :ref:`kafka-source-overview` section.
+
+Security and Authentication
+---------------------------
+
+Learn how to secure communications between MongoDB and the
+connector in the :ref:`kafka-connector-security-auth` section.
```

Comment:
Suggestion:
I think this is a good general description of the goals of the section (duplicates the overview of the SSL/TLS and X.509 certificates page), but could be more helpful to the user if it included a more specific description about the contents of the MONGODB-AWS the section contains. Would like to see what you come up with here!